### ISSUE TITLE
ppoagent summarizer results in error

### ISSUE THREAD

**Issue by lakedai** [tensorforce/tensorforce#549]
HI,
construct a PPOAgent with summarizer  results in errors. 
the error msg is:

NotFoundError (see above for traceback): Resource localhost/logdir:/home/star/test/s/N10tensorflow22SummaryWriterInterfaceE does not exist.
	 [[node ppo/FlushSummaryWriter (defined at /home/star/anaconda3/envs/gym/lib/python3.5/site-packages/tensorforce/models/model.py:314)  = FlushSummaryWriter[_device="/job:localhost/replica:0/task:0/device:CPU:0"](ppo/Summary

the code is:

env = OpenAIGym('LunarLander-v2', visualize=False)

network_spec = [

     dict(type='dense', size=32, activation='tanh'),
     dict(type='dense', size=32, activation='tanh')

]

summary_spec = {

    'directory': '/home/star/test/s',
    'steps': 50,
    'labels': [
        'configuration',
        'gradients_scalar',
        'regularization',
        'inputs',
        'losses',
        'variables'
    ]

}

agent = PPOAgent(

    states=env.states,
    actions=env.actions,
    network=network_spec,
    step_optimizer=dict(
        type='adam',
        learning_rate=1e-3
    ),
    summarizer=summary_spec,
    optimization_steps=10,
    entropy_regularization=0.01,

)

**Comment by AlexKuhnle** [comment]
Hi @lakedai,
Summaries are unfortunately a problem in the current master version of the framework (I think "graph" and "losses" are the only one working). However, the new version, currently under branch `major-revision` does a better job at handling summaries, so you maybe want to give it a try already (warning: still under development and not well documented, but most things should work by now, let me know if they don't). Check out for instance the [summaries test](https://github.com/tensorforce/tensorforce/blob/major-revision/tensorforce/tests/test_summaries.py).

**Comment by lakedai** [comment]
@AlexKuhnle 
thank you. the `major-revision` branch just works perfectly.
